src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy_test.go

```diff
@@ -113,6 +109,9 @@ func TestSshProxy(t *testing.T) {
 	serverSshPeer.Ready()
 
 	s := test.HttpServer("7020")
+	if s == nil {
+		t.Error("Failed - not create server")
```

Comment:
Okay as it's in lots of places I think find and replace will be easier than me commenting everywhere.